src/trees.jl

```diff
@@ -23,8 +25,8 @@ Base.getindex(p::Path, v::AbstractVector) = Path(p.entries[v])
 Base.setindex(p::Path, x, i::Int) = p.entries[i] = x
 
 mutable struct SceneNode
-    object::Nullable{Vector{UInt8}}
-    transform::Nullable{Vector{UInt8}}
+    object::Union{Vector{UInt8}, Nothing}
```

Comment:
This will be slower on 0.6 but I doubt it matters in practice

Reply:
I wasn't sure whether this was being accessed in a hot loop. I could add type assertions if it is.

Reply:
Well, it's accessed any time you do `setobject!` or `settransform!`, but since those involve network IO and async tasks I suspect it's not worth worrying about.